src/Common/DateLUTImpl.cpp

```diff
@@ -284,6 +285,13 @@ namespace cctz_extension
             const std::string & name,
             const std::function<std::unique_ptr<cctz::ZoneInfoSource>(const std::string & name)> & fallback)
         {
+            const char * prefer_system_tzdata_env = std::getenv("PREFER_SYSTEM_TZDATA"); // NOLINT(concurrency-mt-unsafe)
+            if (prefer_system_tzdata_env && 0 != strcmp(prefer_system_tzdata_env, "0"))
```

Comment:
```suggestion
            if (prefer_system_tzdata_env && 
                0 != strcmp(prefer_system_tzdata_env, "0") && 
                0 != strcasecmp(prefer_system_tzdata_env, "false"))
```

I.e. should we ready that somebody will set it to false explicitly?